server/templates/base.html

```diff
@@ -132,6 +133,8 @@ <h6>{% trans %}Explore{% endtrans %}</h6>
             <a href="{{ url_for('tools.scatter') }}">{% trans %}Scatter Plot Explorer{% endtrans %}</a>
             {# TRANSLATORS: The name of a tool to explore maps of statistical variables for places. #}
             <a href="{{ url_for('tools.map') }}">{% trans %}Map Explorer{% endtrans %}</a>
+            {# TRANSLATORS: The name of a tool to explore statistical variables. #}
```

Comment:
please add more details for translators -- it would help to be more descriptive about what the tool does.

Reply:
done